diff --git a/pytype/tools/analyze_project/pytype_runner.py b/pytype/tools/analyze_project/pytype_runner.py
--- pytype/tools/analyze_project/pytype_runner.py
+++ pytype/tools/analyze_project/pytype_runner.py
@@ -1,6 +1,7 @@
 """Use pytype to analyze and infer types for an entire project."""
 
+import os
 import collections
 import itertools
 import logging
 import subprocess
@@ -36,8 +37,10 @@
 
 
 def _get_executable(binary, module=None):
   """Get the path to the executable with the given name."""
+  if binary == "ninja":
+    return [os.path.join(os.environ.get("CONDA_PREFIX", None), "bin", "ninja")]
   if binary == 'pytype-single':
     custom_bin = path_utils.join('out', 'bin', 'pytype')
     if sys.argv[0] == custom_bin:
       # The Travis type-check step uses custom binaries in pytype/out/bin/.
