diff --git a/pytype/tools/analyze_project/pytype_runner.py b/pytype/tools/analyze_project/pytype_runner.py
--- pytype/tools/analyze_project/pytype_runner.py
+++ pytype/tools/analyze_project/pytype_runner.py
@@ -46,9 +46,10 @@
               path_utils.abspath(path_utils.dirname(custom_bin)),
               'pytype-single')
       ])
   if sys.executable is not None:
-    return [sys.executable, '-m', module or binary]
+    if importlib.util.find_spec(binary or binary) is not None:
+      return [sys.executable, '-m', module or binary]
   else:
     return [binary]
 PYTYPE_SINGLE = _get_executable('pytype-single', 'pytype.single')
 
