diff --git a/pytype/tools/analyze_project/pytype_runner.py b/pytype/tools/analyze_project/pytype_runner.py
--- pytype/tools/analyze_project/pytype_runner.py
+++ pytype/tools/analyze_project/pytype_runner.py
@@ -1,6 +1,7 @@
 """Use pytype to analyze and infer types for an entire project."""
 
+import os
 import collections
 import itertools
 import logging
 import subprocess
@@ -45,12 +46,19 @@
           path_utils.join(
               path_utils.abspath(path_utils.dirname(custom_bin)),
               'pytype-single')
       ])
-  if sys.executable is not None:
-    return [sys.executable, '-m', module or binary]
+  elif binary == 'ninja':
+    try:
+      import ninja
+      return [sys.executable, '-m', module or binary]
+    except ImportError:
+      return [binary]
   else:
-    return [binary]
+    if sys.executable is not None:
+      return [sys.executable, '-m', module or binary]
+    else:
+      return [binary]    
 PYTYPE_SINGLE = _get_executable('pytype-single', 'pytype.single')
 
 
 def resolved_file_to_module(f):
